src/sentry/static/sentry/app/components/assigneeSelector.tsx

```diff
@@ -388,9 +387,7 @@ const Label = styled(TextOverflow)`
   margin-left: 6px;
 `;
 
-const ClearAssigneeIcon = styled(props => (
-  <InlineSvg {...props} src="icon-circle-close" />
-))`
+const ClearAssigneeIcon = styled(props => <IconClose {...props} size="xs" isCircled />)`
   ${getSvgStyle};
 `;
```

Comment:
I've realized these days the "better" way to do this (because in this case here, `props` is not typed`) would be as such:

``tsx
const ClearAssigneeIcon = styled(IconClose)`
  ...
`;

ClearAssigneeIcon.defaultProps = {
  isCircled: true,
  size: 'xs',
}
```

Either that or type `props`